### PR TITLE
add notification settings for Moderation extension

### PR DESCRIPTION
**Pull request by robkam**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3600,6 +3600,20 @@
...
+	// Moderation settings
+	wgModerationNotificationEnable' => array( 
+		'default' => false, // Disable notifications
+		'scruffywiki' => true,
+		'sdiywiki' => true,
+	),
+	wgModerationNotificationNewOnly' => array(
+		'default' => false, // Notify administrator only about new pages requests
+	),
+	wgModerationEmail' => array(
```
Comment (Reception123) [inline LocalSettings.php]: Missing ''

File: LocalSettings.php [hunk 2]
```diff
@@ -3600,6 +3600,20 @@
 		'trexwiki' => false,
 	),
 
+	// Moderation settings
+	wgModerationNotificationEnable' => array(
```
Comment (Reception123) [inline LocalSettings.php]: Missing ''

Comment (robkam) [reply LocalSettings.php]: thanks

File: LocalSettings.php [hunk 3]
```diff
@@ -3600,6 +3600,20 @@
...
+		'default' => false, // Disable notifications
+		'scruffywiki' => true,
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array(
+		'default' => false, // Notify administrator only about new pages requests
+	),
+	'wgModerationEmail' => array(
+		'default' =>  $wgEmergencyContact, // Email address to send notifications
+		'sdiywiki' => admin@sdiy.info,
```
Comment (Reception123) [inline LocalSettings.php]: this should also be placed between ''

**Pull request closed without merging**